ember-core/shared/src/main/scala/org/http4s/ember/core/h2/PseudoHeaders.scala

```diff
@@ -162,4 +166,11 @@ private[h2] object PseudoHeaders {
     }.toOption
       .flatten
 
+  val connectionHeadersFields: Set[CIString] = Set(
+    org.http4s.headers.`Transfer-Encoding`.headerInstance.name,
+    org.http4s.headers.Connection.headerInstance.name,
+    org.http4s.headers.Upgrade.headerInstance.name,
+    org.http4s.headers.`Keep-Alive`.headerInstance.name,
+    CIString("Proxy-Connection"),
+  )
```

Comment:
Maybe add a comment linking to the RFC here?
https://httpwg.org/specs/rfc7540.html#rfc.section.8.1.2.2

Reply:
Added